sewer/auth_providers/tests/test_utils.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+
+class MockResponse(object):
+    """
+    mock python-requests Response object
+    """
+
+    def __init__(self, status_code=200, content=None):
+        if not content:
+            content = {}
+
+        try:
+            content.update(
+                {
+                    "something": "ok",
+                    "result": [{"name": "example.com", "id": "some-mock-dns-zone-id"}],
+                }
+            )
+        except AttributeError:
+            json_loads = json.loads
+        else:
+            json_loads = lambda a: a
+
+        self.content = json.dumps(content).encode()
+        self.text = json_loads(self.content.decode())
+
+        self.headers = {}
+        self.status_code = status_code
+
+    def json(self):
+        return json.loads(self.content.decode())
+
+
+class mockLibcloudDriverZone(object):
+    """
+    A mock of a dns zone in a libcloud drivers dns
+    """
+
+    id = "mock-zone-id-1"
+
+    def create_record(self, name, type, data):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'type'](https://app.codacy.com/manual/komuW/sewer/pullRequest?prid=5264928)